AraCorrelator/RayTraceCorrelator_detail.h

```diff
@@ -16,6 +16,20 @@ double RayTraceCorrelator::fastEvalForEvenSampling(TGraph * grIn, double xvalue)
     return FFTtools::simpleInterploate(xVals[p0], yVals[p0], xVals[p0 + 1], yVals[p0 + 1], xvalue);
 }
 
+TGraph* RayTraceCorrelator::getNormalisedGraph(TGraph *grIn){
```

Comment:
Minutia, but since there are a lot of ways one can normalize something, would it be helpful to add to the function name that this graph is normalized by RMS?

Reply:
Good thought! I made that change. See if what you think now?